terraform/ecr_webserver.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_ecr_repository" "webserver_nginx" {
+  name                 = "webserver-nginx"
+  image_tag_mutability = "MUTABLE"
+
+  image_scanning_configuration { scan_on_push = false }
```

Comment:
What was the reason for disabling scanning images on push? It is a security feature, right?

Reply:
It is. But:
- Its slow and costs (although not much!)
- It's going to to tell me every time a package on any machine needs updating
- I can't imagine I'm acutally ever going to go and look at it.

But maybe we set it up so it actually alerts us etc, and then it would be valuable once we're in production.

Reply:
Okay cool.